src/java.base/share/classes/java/net/URL.java

```diff
@@ -129,7 +129,9 @@
  * the protocol, host name, or port number is missing, the value is
  * inherited from the fully specified URL. The file component must be
  * specified. The optional fragment is not inherited.
- * <p>
+ *
+ * <h2><a id="constructor-deprecation"></a>Constructing instances of {@code URL}</h2>
```

Comment:
Would it be better to move the anchor to line 164 (the line where it says that the URL constructors are deprecated?

Reply:
To be discussed: I actually wanted the deprecation link ( the link from `@deprecated` ) to lead here because I find that the whole section is relevant for developers who might want to decide whether to actually move away from using constructors, or be tempted to just use `@SuppressWarnings`.

Reply:
Actually... Maybe I could move up the paragraph that says that URL constructors are deprecated up here, just after the <h2> title? Would that be better?

Reply:
Try it, it might be better. I think the main thing is that link brings the reader to somewhere close to the deprecated message.

Reply:
Done.